src/musicbrainz/gzip.cpp

```diff
@@ -40,7 +40,7 @@ QByteArray gzipCompress(const QByteArray &data)
 	compressedData.remove(compressedData.size() - 4, 4); // zlib footer
 
 	QByteArray result;
-	result.append(header, 10);
+	result.append(reinterpret_cast<const char*>(header), 10);
```

Comment:
reinterprete_cast looks evil, use static_cast in such a case. 
However we have append(const char * str) so the original version was not that bad. 
What was the error message? Probably that the array initializer are int.  

One other problem is that char can either be signed or unsigned. It is compiler specific. 

http://en.cppreference.com/w/cpp/language/character_literal

According to this a cast free solution will look like: 

```
	const unsigned char header[10] = {
		'\x1f', '\x8b', // ID1 + ID2
		'\8', // Compression Method
		'\0', // Flags
		'\0', '\0', '\0', '\0', // Modification Time
		'\2', // Extra Flags
		'\255', // Operating System
	};
```
Can you try this?